integreat_cms/cms/templates/events/event_form.html

```diff
@@ -142,6 +142,10 @@ <h3 class="heading font-bold text-black">
                     </div>
                     <div class="w-full p-4 rounded bg-white">
                         <div>
+                            <label>{% trans 'Maximal duration allowed for one-time event is 7 days' %}</label>
+                            <div class="help-text">
+                                {% trans 'Set an event longer than 7 days as recurring event.' %}
+                            </div>
```

Comment:
```suggestion
```
I don't think we need this as permanent info - the warning when somebody tries to save an event with more than 7 days should be enough.